### DISCUSSION TITLE
Adding Translated Documents

### DISCUSSION THREAD

**Discussion by yamatoya** [statiqdev/Discussions#23] [Q&A] [answered]
Can I add localized document pages? For example, imagine that the site has a Japanese translation of the documentation under "ja-jp".
If possible, I would like to provide Japanese translations of the documentation.

**Comment by daveaglick** [comment, marked as answer]
There's nothing in-the-box right now, though you could certainly add folders like `input/ja-jp` directly and the files inside would output to that location. Another approach I've seen is to load JSON files with various translations and locales:

`en-us.json`:
```json
{
  "foo": "buzz"
}
```

`ja-jp.json`:
```json
{
  "foo": "bazz"
}
```

Then use those values inside a Razor template with something like `Outputs["ja-jp.json"].First().GetString("Foo")`. A more sophisticated setup could even put the translation files in a subdirectory, iterate on them, and produce an output document for each one. This approach works well when you've got common content with just a handful of things that need to be translated.

Eventually Statiq will get some built-in localization support, I'm just not sure yet when that'll be or what it'll look like.
